[Caggs] Update troubleshooting guide (#1216)
timescale/docs: _partials/_caggs-function-support.mdx

> The following table summarizes aggregate function support in continuous

Reviewer: ```suggestion
This table summarizes aggregate function support in continuous
```

aggregates:

> |Function, clause, or feature|TimescaleDB 2.7 and above|Below TimescaleDB 2.7|

Reviewer: Can we change the version columns around here? Make the left column `Timescale 2.6 and earlier` and the right-most column `TimescaleDB 2.7 and later`?

Reviewer: (I appreciate that this is a drive by, but I hadn't noticed how awkward it was in previous edits 😕 )
